modules/serverless-create-func-kn.adoc

```diff
@@ -22,23 +22,23 @@ Before you can build and deploy a function, you must create it by using the Knat
 ----
 $ kn func create -r <repository> -l <runtime> -t <template> <path>
 ----
-** Accepted runtime values include `node`, `go`, `python`, `quarkus`, and `typescript`.
-** Accepted template values include `http` and `events`.
+** Accepted runtime values include `quarkus`, `node`, `typescript`, `go`, `python`, `springboot`, and `rust`.
++
+:FeatureName: `node`, `typescript`, `go`, `python`, `springboot`, and `rust` function runtimes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+** Accepted template values include `http` and `cloudevents`.
 +
 .Example command
 [source,terminal]
 ----
-$ kn func create -l typescript -t events examplefunc
+$ kn func create -l typescript -t cloudevents examplefunc
 ----
 +
 .Example output
 [source,terminal]
 ----
-Project path:  /home/user/demo/examplefunc
-Function name: examplefunc
-Runtime:       typescript
-Template:      events
-Writing events to /home/user/demo/examplefunc
+Created typescript function in /home/user/demo/examplefunc
 ----
 +
 ** Alternatively, you can specify a repository that contains a custom template.
```

Comment:
For the template example the output also has changed (just as the above change in L41).

So the lines 55 to 59 should be replaced by
```
Created node function in /home/user/demo/examplefunc
```